### DISCUSSION TITLE
Visually edit the generated graph to make it look nicer/ cleaner

### DISCUSSION THREAD

**Discussion by wout-junius** [mermaid-js/mermaid#4239] [Q&A] [answered]
I was wondering if there is a way to change the graph order so that lines don't overlap or is clearer to us.
thereof making using it in documents way better.

Is there a way to import it into something like draw.io or Lucidcharts so I could edit it there?

I love using mermaid for the quick creation of diagrams but don't always like the outcome and would want to tweak the visuals.

For example the 2 lines going to `WRITE_ACCES` do not need to overlap.
<img width="268" alt="image" src="https://user-images.githubusercontent.com/17813748/227497603-8dd4cd0b-501c-4387-9147-6b90f7aec5d2.png">

**Comment by Yokozuna59** [comment, marked as answer]
Not sure about LucidCharts, but it's possible with draw.io. From https://drawio-app.com/blog/create-mermaid-diagrams-in-draw-io/:

> 1. Select the draw.io macro in your Confluence by typing / which brings up the macros menu.
> 2. Choose a blank diagram and don’t forget to give your new diagram a name.
> 3. Go to the toolbar and select the + symbol.
(You can also insert a mermaid diagram using the menu bar: Arrange > Insert > Advanced > Mermaid)
> 4. Select Advanced and then Mermaid.
> 5. A dialog box will pop up where you can enter the text for the diagram you want to create.
